packages/core/src/bin/commands/start.ts

```diff
@@ -52,9 +52,16 @@ export async function start({ cliOptions }: { cliOptions: CliOptions }) {
   const cleanup = async () => {
     await cleanupReloadable();
     await cleanupReloadableServer();
+    if (listenConnection) {
+      if (listenConnection.dialect === "postgres") {
+        listenConnection.connection.release();
+      }
+    }
```

Comment:
Why is listenConnection not a property of `database` ? They seem to be passed around together.